webapp/admin/views.py

```diff
@@ -42,6 +43,24 @@ def get_stores():
     )
 
 
+@admin.route("/admin/<store_id>/snaps/search")
```

Comment:
```suggestion
@admin.route("/admin/<store_id>/snaps/search.json")
```

I like when we add the .json at then, it helps understanding what am I going to get as a return value